server/protocols/webadmin-integration-test/pom.xml

```diff
@@ -27,162 +28,22 @@
         <relativePath>../../pom.xml</relativePath>
     </parent>
 
-    <artifactId>james-server-webadmin-integration-test</artifactId>
-    <packaging>jar</packaging>
+    <artifactId>webadmin-integration-test</artifactId>
+    <packaging>pom</packaging>
+    <name>Apache James :: Server :: RabbitMQ :: Web Admin server integration tests</name>
```

Comment:
`<name>Apache James :: Server :: Web Admin server integration tests</name>`

Reply:
TODO: fix it in part 1